### PR TITLE
codespaces-linux v2 image

### PR DESCRIPTION
**Pull request by samruddhikhandale**
- [x] Update the oryx image tag consisting of these [proposal changes](https://github.com/github/codespaces/issues/7171#issuecomment-1081080810)
- [x] Remove languages/frameworks from the codespace base image with these [proposal changes](https://github.com/github/codespaces/issues/7171#issuecomment-1081221878)
- [x] Bump up the `codespaces-linux` version to `2.0.0`
- [x] Add python packages. #7205

### CODE REVIEW

**Comment by samruddhikhandale** [comment]
Smoke tests in this PR are failing similar to https://github.com/microsoft/vscode-dev-containers/pull/1255#issuecomment-1017737126

> ❌ oryx-install-dotnet-3.1.0 check failed.
❌ dotnet-3.1.0-installed check failed.

>❌ oryx-build-jekyll check failed.

File: containers/codespaces-linux/history/dev.md [hunk 1]
```diff
@@ -113,15 +101,14 @@ mcr.microsoft.com/vscode/devcontainers/universal:dev
 | [yarn](https://yarnpkg.com/) | 1.22.15 | /opt/yarn |
 | [Maven](https://maven.apache.org/) | 3.6.3 | /opt/maven |
 | [Gradle](https://gradle.org/) | 7.4.2 | 
-| Docker (Moby) CLI &amp; Engine | 20.10.14+azure | 
+| Docker (Moby) CLI &amp; Engine | 20.10.14 |
```
Comment (joshspicer) [inline containers/codespaces-linux/history/dev.md:104]: This file is auto-generated (so nbd), but this `+azure` is metadata showing that we installed moby from the microsoft azure repo (not related to the azure cli, which I suspect is why this was removed)

**Comment by joshspicer** [comment]
Looks great to me @samruddhikhandale! The "smoke test" there passed on all the important tests (those three failures are persistent bugs with the upstream oryx image and file ownership specifically with the smoketest)

File: containers/codespaces-linux/history/dev.md (hunk 1 above)
Comment (samruddhikhandale) [reply containers/codespaces-linux/history/dev.md:104]: ah! my bad. I'll revert this file changes 👍

**Pull request merged**